raitools/requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy>=1.17.2
+pandas>=0.25.1
+scipy>=1.4.1
+scikit-learn>=0.22.1
+lightgbm>=3.1.1
+interpret-community>=0.17.0
+fairlearn>=0.5.0
```

Comment:
the fairlearn dependency should be compatible with azuremlcli.

Reply:
the hope is that azuremlcli will upgrade by the time this is made a dependency

Reply:
Can we still keep this at 0.4.6 as in azuremlcli and upgrade once azuremlcli upgrade?

Reply:
ok done!